File: AccommodationModule/src/main/java/ftn/xmlws/dto/AccommodationUnitsDTO.java
package ftn.xmlws.dto;

import java.util.ArrayList;
import java.util.List;

public class AccommodationUnitsDTO {

	private List<AccommodationUnitDTO> accommodationUnits;
	
	public AccommodationUnitsDTO() {
		this.accommodationUnits = new ArrayList<>();
	}

	public List<AccommodationUnitDTO> getAccommodationUnits() {
		return accommodationUnits;
	}

	public void setAccommodationUnits(List<AccommodationUnitDTO> accommodationUnits) {
		this.accommodationUnits = accommodationUnits;
	}
}

File: xmlws-front/src/app/dto/AccommodationUnitsDTO.ts
import { AccommodationUnitDTO } from './AccommodationUnitDTO';
import { Observable } from '../../../node_modules/rxjs';

export class AccommodationUnitsDTO {
    _accommodationUnits : AccommodationUnitDTO[] = [];

    get accommodationUnits() : AccommodationUnitDTO[] {
        return this._accommodationUnits;
    }
}